components/sup/src/manager/mod.rs

```diff
@@ -48,16 +49,16 @@ use serde;
 use serde_json;
 use time::{self, Timespec, Duration as TimeDuration};
 
-pub use manager::service::{Service, ServiceConfig, ServiceSpec, UpdateStrategy, Topology};
-use self::service::{DesiredState, StartStyle};
+pub use self::service::{Service, ServiceSpec, UpdateStrategy, Topology};
```

Comment:
I guess this was part of our unspoken conventions that we're taking from Aaron Turon's [Rust Guidelines](https://aturon.github.io/style/imports.html) book.